src/picosvg/svg.py

```diff
@@ -381,111 +475,155 @@ def _combine_clip_paths(self, clip_paths: Sequence[SVGPath]) -> SVGPath:
             return clip_paths[0]
         return SVGPath.from_commands(intersection(clip_paths))
 
-    def _new_id(self, tag, template):
-        for i in range(100):
+    def _new_id(self, template):
+        for i in range(256):
```

Comment:
what if we have more than 256?

Reply:
It was pretty arbitrary ... I can make it substantially larger w/o issue.

Reply:
1 << 16 ? ... I just like having an exit rather than looping forever

Reply:
Fixed

Reply:
even if 65k may seem reasonable now, to me it still feels like an unnecessary restrictions. There's no risk you'll end up in an infinite loop here. It would be better to keep it unbounded, e.g. use `itertools.count` iterator
https://docs.python.org/3/library/itertools.html#itertools.count

but whatever, I don't really mind either way.